src/main/scala/com/microsoft/hyperspace/index/IndexLogEntryTags.scala

```diff
@@ -16,9 +16,50 @@
 
 package com.microsoft.hyperspace.index
 
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.catalyst.plans.logical.LogicalPlan
+
+import com.microsoft.hyperspace.util.HyperspaceConf
+
 object IndexLogEntryTags {
-  // HYBRIDSCAN_REQUIRED indicates if Hybrid Scan is required for this index or not.
+  // HYBRIDSCAN_REQUIRED indicates if Hybrid Scan is required for the index or not.
   // This is set in getCandidateIndexes and utilized in transformPlanToUseIndex.
   val HYBRIDSCAN_REQUIRED: IndexLogEntryTag[Boolean] =
     IndexLogEntryTag[Boolean]("hybridScanRequired")
+
+  // IS_SIGNATURE_MATCH indicates if the plan has the same signature value with the index or not.
+  val IS_SIGNATURE_MATCH: IndexLogEntryTag[Boolean] =
+    IndexLogEntryTag[Boolean]("isSignatureMatch")
+
+  // IS_HYBRIDSCAN_CANDIDATE indicates if the index can be applied to the plan using Hybrid Scan.
+  val IS_HYBRIDSCAN_CANDIDATE: IndexLogEntryTag[Boolean] =
+    IndexLogEntryTag[Boolean]("hybridScanCandidate")
+
+  // IS_HYBRIDSCAN_CANDIDATE indicates if the index can be applied to the plan using Hybrid Scan.
+  val HYBRIDSCAN_CONFIG_CAPTURE: IndexLogEntryTag[Seq[String]] =
+    IndexLogEntryTag[Seq[String]]("hybridScanConfigCapture")
+
+  private def getHybridScanConfigs(spark: SparkSession): Seq[String] = {
+    Seq(
+      HyperspaceConf.hybridScanEnabled(spark).toString,
```

Comment:
Note to reviewers: In fact this is not necessary as the hybrid scan related tags are referred only if hybrid scan is enabled.
I added this for the context of the function name.

Reply:
Should `getHybridScanRelatedConfigs` belong to `IndexLogEntryTags`? Why not define this where it's used? We can make a generic version if required later.